### PR TITLE
This patch will BLOW your mind..

### PR DESCRIPTION
**Pull request by aabmass**

### CODE REVIEW

**Comment by aabmass** [comment]
Please merge this request *asap*...

**Comment by aabmass** [comment]
This PR has important `bug fixes`

File: src/common/components/Home.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ class Home extends Component {
       	<div className="posts">
   
   			<div className="post banner">
-			    <h1 className="post-title">I build <em>scalable</em>, <em>maintainable</em> and <em>secure</em> front-end and full-stack web applications.</h1>
+			    <h1 className="post-title">I build <em>scalable</em>, <em>maintainable</em> and <em>secure</em> front-end and full-stack web applications. These use only <em>dynamic programming</em> to <em>DP</em> you where it hurts...</h1>
```
Comment (aabmass) [inline src/common/components/Home.js]: Here is your `problem`

**Review by luisevilleda — CHANGES_REQUESTED** [review]
These changes are trash

**Comment by aabmass** [comment]
I am deeply TRIGGERED

**Comment by aabmass** [comment]
@luisevilleda #1 Me pones muy triste

**Pull request closed without merging**